src/Controllers/CopyPatrol.php

```diff
@@ -357,24 +357,22 @@ protected function getRecords() {
 		$lastId = $this->request->get( 'lastId' ) ?: 0;
 		$drafts = $this->request->get( 'drafts' ) ? '1' : null;
 		$wikiprojects = null; // for the server and use in SQL
-		$wikiprojectsArray = []; // for the clientside and showing <option>s in Select2 control
-		// account for empty URL param, e.g. wikiprojects= (no value set)
-		if ( $this->request->get( 'wikiprojects' ) && $this->request->get( 'wikiprojects' ) !== '' ) {
-			// WikiProjects are submitted like 'Medicine|Military_History|Something'
-			// This is to be URL and Select2-friendly, and use the
-			//   standard pipe-separated titles we see in MediaWiki
-			$wikiprojects = $this->request->get( 'wikiprojects' );
-			$wikiprojectsArray = explode( '|', $wikiprojects );
-		}
+		$searchText = $this->request->get( 'searchText' );
+		$searchCriteria = $this->request->get( 'searchCriteria' );
+
 		// make this easier when working locally
 		$numRecords = $_SERVER['HTTP_HOST'] === 'localhost' ? 10 : 50;
+
 		// compile all options in an array
 		$options = [
 			'filter' => $filter,
 			'last_id' => $lastId > 0 ? $lastId : null,
 			'drafts' => $drafts,
-			'wikiprojects' => $wikiprojects
+			'wikiprojects' => $wikiprojects,
```

Comment:
Are we still supporting wikiprojects (outside showing them in the WikiProjects column)? I guess we could leave the code there just in case we want to revive it.